chainerx_cc/chainerx/python/error.cc

```diff
@@ -20,6 +20,14 @@ void InitChainerxError(pybind11::module& m) {
     py::register_exception<NotImplementedError>(m, "NotImplementedError");
     py::register_exception<GradientError>(m, "GradientError");
     py::register_exception<GradientCheckError>(m, "GradientCheckError");
+
+    py::register_exception_translator([](std::exception_ptr p) {
+        try {
+            if (p) std::rethrow_exception(p);
+        } catch (const IndexError& e) {
+            PyErr_SetString(PyExc_IndexError, e.what());
```

Comment:
How about the following, I think it's easier to read at least and the effect should be the same.
```suggestion
            throw py::index_error{e.what()};
```

Reply:
Fixed.